op-service/sources/supervisor_client.go

```diff
@@ -114,6 +114,15 @@ func (cl *SupervisorClient) Finalized(ctx context.Context, chainID types.ChainID
 	return result, err
 }
 
+func (cl *SupervisorClient) FinalizedL1(ctx context.Context) (eth.BlockID, error) {
```

Comment:
This should return an `eth.BlockRef`, since that's what the RPC server is serving. They're compatible, ID is a subset of a regular blockref, but I think they should just be consistent.